<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetFunc</class>
 <widget class="QWidget" name="WidgetFunc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>37</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>37</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>360</width>
    <height>37</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_value">
   <property name="geometry">
    <rect>
     <x>61</x>
     <y>0</y>
     <width>211</width>
     <height>37</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	background-color: rgb(52, 53, 56);
	border-top: 1px  solid rgb(150, 150, 150) ;
	border-left: 0px  solid rgb(150, 150, 150) ;
	border-bottom: 1px  solid rgb(150, 150, 150) ;
	border-right: 1px  solid rgb(150, 150, 150) ;
}</string>
   </property>
   <property name="text">
    <string>pi = 3.1415 ыыыыы</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_delete">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>0</y>
     <width>31</width>
     <height>37</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-radius: 0px;
	background-color: rgb(52, 53, 56);
	border-top: 1px  solid rgb(150, 150, 150) ;
	border-left: 1px  solid rgb(150, 150, 150) ;
	border-bottom: 1px  solid rgb(150, 150, 150) ;
	border-right: 0px  solid rgb(150, 150, 150) ;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_type">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>31</width>
     <height>37</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-radius: 5px;
	border: 1px  solid rgb(150, 150, 150) ;
	background-color: rgb(95, 99, 104);
}

 QPushButton:pressed {
	border-radius: 5px;
	border: 1px  solid rgb(150, 150, 150) ;
	background-color: rgb(115, 119, 124);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_hide">
   <property name="geometry">
    <rect>
     <x>272</x>
     <y>0</y>
     <width>43</width>
     <height>37</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-radius: 5px;
	border: 1px  solid rgb(150, 150, 150) ;
	background-color: rgb(95, 99, 104);
}

 QPushButton:pressed {
	border-radius: 5px;
	border: 1px  solid rgb(150, 150, 150) ;
	background-color: rgb(115, 119, 124);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_color">
   <property name="geometry">
    <rect>
     <x>314</x>
     <y>0</y>
     <width>43</width>
     <height>37</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-radius: 5px;
	border: 1px  solid rgb(150, 150, 150) ;
	background-color: rgb(95, 99, 104);
}

 QPushButton:pressed {
	border-radius: 5px;
	border: 1px  solid rgb(150, 150, 150) ;
	background-color: rgb(115, 119, 124);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
